hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -202,12 +200,11 @@ protected Process startProcess(ExecutorDriver driver, Task task) {
     reloadConfig();
     if (proc == null) {
       try {
-        ProcessBuilder processBuilder = new ProcessBuilder("sh", "-c", task.getCmd());
-        processBuilder.environment().putAll(createHdfsNodeEnvironment(task));
-        proc = processBuilder.start();
-        procWatcher.watch(proc);
-        task.setProcess(proc);
-        redirectProcess(task.getProcess());
+        Map<String, String> envMap = createHdfsNodeEnvironment(task);
+
+        Process process = ProcessUtil.startCmd(envMap, task.getCmd());
+        procWatcher.watch(process);
+        task.setProcess(process);
```

Comment:
This process in the "Task" stuff is bad.  Again, probably not something that needs to be addressed in this PR.